### PR TITLE
[docusaurus] Fix edit url

### PR DESCRIPTION
**Pull request by gregnazario**
### Checklist

- [x] Do all Lints pass?
- [x] Have you ran `pnpm spellcheck`?
- [x] Have you ran `pnpm fmt`?
- [x] Have you ran `pnpm lint`?

### CODE REVIEW

**Review by moonclavedev — COMMENTED** [review]
Coincidently I was just going to push a PR up for this today myself!

However, my planned url was: `https://github.com/aptos-labs/developer-docs/edit/main/docs/` which I believe is correct, no?

**Comment by gregnazario** [comment]
> Coincidently I was just going to push a PR up for this today myself!
> 
> 
> 
> However, my planned url was: `https://github.com/aptos-labs/developer-docs/edit/main/docs/` which I believe is correct, no?

Unsure, I'm just following the original URL.  Would have to test it out

File: docusaurus.config.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const config = {
           sidebarPath: require.resolve("./sidebars.js"),
           sidebarCollapsible: false,
           editUrl:
-            "https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/",
+            "https://github.com/aptos-labs/developer-docs/",
```
Comment (davidiw) [inline docusaurus.config.js:37]:
```suggestion
            "https://github.com/aptos-labs/developer-docs/edit/main",
```

**Review by davidiw — COMMENTED** [review]
you can test by using the netlify functionality

**Review by davidiw — APPROVED** [review]

**Pull request merged**